packages/ckeditor5-table-resize/src/tablecolumnresize/utils.js

```diff
@@ -42,8 +45,9 @@ export function getAffectedTables( changes ) {
 //
 // @private
 // @param {module:engine/model/differ~DiffItem} changes
+// @param {module:engine/model/model~Model} model
 // @returns {module:engine/model/element~Element|null}
-function getAffectedTable( change ) {
```

Comment:
This function no longer matches to its name. It says that it returns a **table** - while it returns multiple tables. And there's a `getAffectedTables()` right next to it.

Reply:
I think that there's no longer need to have a separation for both functions and it should be just a single function (`getAffectedTables`).